DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+* Updating period-scoped metadata (e.g. codec configuration or DRM signaling).
+
+Periods are self-contained - a service SHALL NOT require a client to know the contents of another period in order to correctly present a period. Knowledge of the contents of different periods MAY be used by a client to achieve seamless period transitions, especially when working with [[#timing-connectivity|period-connected representations]].
+
+All periods SHALL be consecutive and non-overlapping. A period MAY have a duration of zero.
+
+Note: A period with a duration of zero might, for example, be the result of ad-insertion logic deciding not to insert any ad.
+
+<div class="example">
+The below MPD example consists of two 20-second periods. The duration of the first period is calculated using the start point of the second period. The total duration is 40 seconds.
+
+<xmp highlight="xml">
+<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" type="static">
+	<Period>
+		...
+	</Period>
+	<Period start="PT20S" duration="PT20S">
+		...
+	</Period>
+</MPD>
+</xmp>
+
+Parts of the MPD structure that are not relevant for this chapter have been omitted - this is not a fully functional MPD file.
+</div>
+
+In a static MPD, the first period SHALL start at the zero point of the [=MPD timeline=]. In a dynamic MPD, the first period SHALL start at or after the zero point of the [=MPD timeline=].
+
+In a static MPD, the last period SHALL have a `Period@duration`. In a dynamic MPD, the last period MAY have a `Period@duration`, in which case it is considered to have a fixed duration. If without `Period@duration`, the last period in a dynamic MPD SHALL be considered to have an unlimited duration.
+
+Note: In a dynamic MPD, a period with an unlimited duration may be converted to fixed-duration by an MPD update. Periods in a dynamic MPD may also be shortened or removed entirely under certain conditions. See [[#timing-mpd-updates]].
+
+`MPD@mediaPresentationDuration` MAY be present. If present, it SHALL accurately match the duration between the zero point on the MPD timeline and the end of the last period. Clients SHALL calculate the total duration of a static MPD by adding up the durations of each [=period=] and SHALL NOT rely on the presence of `MPD@mediaPresentationDuration`.
+
+Note: This calculation is necessary because the durations of [[#timing-xlink|XLink periods]] can only be known after the XLink is resolved. Therefore it is impossible to always determine the total MPD duration on the service side as only the client is guaranteed to have access to all the required knowledge.
+
+## Representations ## {#timing-representation}
+
+A <dfn>representation</dfn> is a sequence of <dfn>segment references</dfn> and a description of the samples within the referenced [=media segments=]. Each representation belongs to exactly one adaptation set and to exactly one [=period=], although [[#timing-connectivity|a representation may be connected with a representation in another period]].
```

Comment:
> Each representation belongs to exactly one adaptation set...

From @haudiobe: This is not correct. There is no reason that the same physical Representation is not present in multiple Adaptation Sets.

Reply:
This contribution defines a representation as a set of segment references, not as some files on disk. I agree that the same files could be used multiple times. However, in the context of this contribution that makes them different representations.

From my perspective, whether some files are shared or not is irrelevant to DASH clients and services, so should play no role in what defines a representation. If MPEG-DASH takes that view, that is just an example of MPEG-DASH being confusing and requiring reinterpretation in a more human-readable form.